test/transforms.jl

```diff
@@ -131,6 +131,15 @@
     rtₒ = revert(T, n, c)
     @test rt == rtₒ
 
+    # different columntypes
+    tt = (;a=rand(3), b=rand(ComplexF64, 3))
```

Comment:
```suggestion
    t = (; a = rand(3), b = rand(ComplexF64, 3))
```
Change from `tt` to `t` and ajust code style.